.idea/bowling-challenge.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Possibly some of the folders / files in you PR were committed by  accident? e.g '.idea/*`, 'lib/jasmine-3.6.0/*` etc

To ignore stuff you don't intend to commit you can create a `.gitignore` file and add folders or files that you do not want too commit. 

In the root of your folder ...
```
$ touch .gitignore 
$ echo .idea >> .gitignore 
```